forLim.py

```diff
@@ -227,13 +224,21 @@ def initGui(self):
         QObject.connect(self.dlg.LE_polylines, SIGNAL("editingFinished()"), self.check_polylines_path)
         
         
+        ## Select Method Menu
+        #Set convolution as default
+        self.dlg.radio_TR.setChecked(True)
+        
+        #Select method
+        QObject.connect(self.dlg.radio_TR, SIGNAL("clicked()"), self.select_tr_method)
+        QObject.connect(self.dlg.radio_FC, SIGNAL("clicked()"), self.select_fc_method)
+        
         ## Set current window and widget when opening the plugin
         #Set current widget
         self.dlg.tabWidget.setCurrentIndex(0)
```

Comment:
FYI, when you save the ui file in QTCreator, the tabindex is saved as you see it. So, that's a good idea to define it manually here